lib/computer_player.rb

```diff
@@ -0,0 +1,24 @@
+require_relative "player"
+
+class ComputerPlayer < Player
+
+  attr_reader :player_input
+
+  def get_input
+    @display.present(Messages.new.player_prompt(@name))
```

Comment:
This is also in both subclasses. You should look into what the keyword 'super' does.